NOTES.md

```diff
@@ -1,3 +1,186 @@
+# Release 2.0.0-RC1 (Musashi)
+
+Released on 2015-03-23
+
+This is a release candidate for the upcoming RethinkDB 2.0 release.
+It is not for production use. It might still have bugs. If you decide
+to test it, please [back up all][backup-docs] your data first.
```

Comment:
Could we add a second sentence (new paragraph) asking people to report issues on GitHub or the Google Group, if they'd like to be involved with testing the RC?